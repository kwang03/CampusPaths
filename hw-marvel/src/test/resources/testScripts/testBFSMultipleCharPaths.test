#Tests the BFS on a graph where there is more than one path between two characters using different characters on the path

LoadGraph myGraph multipleCharPaths.csv

FindPath myGraph Char1 Char4