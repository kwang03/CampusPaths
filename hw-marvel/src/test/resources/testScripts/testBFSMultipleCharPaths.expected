#Tests the BFS on a graph where there is more than one path between two characters using different characters on the path

loaded graph myGraph

path from Char1 to Char4:
Char1 to Char2 via Book1
Char2 to Char4 via Book2